models/stg_recurly__invoice_history.sql

```diff
@@ -34,7 +34,7 @@ final as (
         collection_method,
         po_number,
         net_terms,
-        UPPER(currency) as currency,
+        UPPER(currency) as currency, 
```

Comment:
Why UPPER?

Reply:
same question for other instances

Reply:
@fivetran-joemarkiewicz Can't tell you why, this is Dimos's code. Didn't want to get rid of it until you had a look at it, figured it might be some accounting thing? But some other currencies aren't UPPERed so not sure what's going on. I've removed it now.

Reply:
@fivetran-joemarkiewicz & @fivetran-avinash that's because of [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) where all currency codes are written in uppercase.